tests/gtest/avif_fuzztest_helpers.h

```diff
@@ -226,10 +226,11 @@ inline auto ArbitraryBaseAvifDecoder() {
 // options fuzzed, with the exception of 'ignoreColorAndAlpha' (because it would
 // break most tests' assumptions).
 inline auto ArbitraryAvifDecoderWithGainMapOptions() {
-  return fuzztest::Map(
-      AddGainMapOptionsToDecoder, ArbitraryBaseAvifDecoder(),
-      /*enable_parsing_gain_map_metadata=*/fuzztest::Arbitrary<bool>(),
-      /*enable_decoding_gain_map=*/fuzztest::Arbitrary<bool>());
+  return fuzztest::Map(AddGainMapOptionsToDecoder, ArbitraryBaseAvifDecoder(),
+                       /*gain_map_decode_mode=*/
```

Comment:
Nit/optional: I tend not to add the name of the field when the values are explicit enough